index.html

```diff
@@ -10,392 +10,422 @@
   </head>
 
   <body>
-  <div id="main">
 
     <!--1) GREEN BACKGROUND START-->
     <div class="bg-green">
 
-    <!--HEADER-->
-      <div class="content-wrap lrg-rl-padding" id="header">
-        <header>
-          <p class="text -x-small -color-dark -caps -wide-spacing -bold centered">A Viget Exploration</p>
-          <img class="header-img" src="./public/img/title.png" alt="Improving Medical Adherence">
-        </header>
-        <div class="pills -yellow"></div>
-        <div class="pills -multiple"></div>
-        <div class="pills -double"></div>
-        <div class="pills -purple"></div>
-      </div>
-
-
-    <!--INTRO SECTION-->
-      <div class="content-wrap lrg-rl-padding bg-white" id="intro">
-        <section>
+      <!--HEADER-->
+      <header class="header">
+        <div class="content-wrap lrg-rl-padding"> <!--content-wrap is for padding-->
+          <h1 class="text -x-small -color-dark -caps -wide-spacing -bold -centered">
+            A Viget Exploration
+            <img class="header__img" src="./public/img/title.png" alt="Improving Medical Adherence">
+          </h1>
+
+          <div class="pills -yellow"></div>
+          <div class="pills -multiple"></div>
+          <div class="pills -double"></div>
+          <div class="pills -purple"></div>
+        </div>
+      </header>
+
+
+      <!--INTRO SECTION-->
+      <section class="intro bg-white">
+        <div class="content-wrap lrg-rl-padding">
```

Comment:
☝️ yep, a great way to add additional information for screen readers. This suggestion will make more sense when we do a demo on Thursday.